examples/tests/transform.cpp

```diff
@@ -46,3 +43,15 @@ TEST_F(TransformAlgorithm, TestStdTransform) {
 
   EXPECT_TRUE(ranges::equal(expected, vb));
 }
+
+TEST_F(TransformAlgorithm, TestStdTransform) {
+  test_with_size(1024);
+}
+
+TEST_F(TransformAlgorithm, TestSmallSize) {
+  test_with_size(16);
+}
+
+TEST_F(TransformAlgorithm, TestOddSize) {
+  test_with_size(555);
+}
```

Comment:
Use value parameterized tests instead (https://github.com/google/googletest/blob/master/googletest/docs/AdvancedGuide.md#value-parameterized-tests)

Reply:
Thanks for the link!